src/App.js

```diff
@@ -1,26 +1,16 @@
 import React, { Component } from 'react';
-import logo from './logo.svg';
 import './App.css';
+import Bounty from './Components/Bounty';
+
 
 class App extends Component {
   render() {
     return (
-      <div className="App">
-        <header className="App-header">
-          <img src={logo} className="App-logo" alt="logo" />
-          <p>
-            Edit <code>src/App.js</code> and save to reload.
-          </p>
-          <a
-            className="App-link"
-            href="https://reactjs.org"
-            target="_blank"
-            rel="noopener noreferrer"
-          >
-            Learn React
-          </a>
-        </header>
-      </div>
+      <Bounty>
+        <Bounty.BountyTitle></Bounty.BountyTitle>
```

Comment:
I wouldnt render these components like this in your app. what does your bounty contain? a title, info, and reward? Your bounty component should contain these components. and the app should only render the bounty. bounty/index.js should then contain all of your other components which you have in your divs. I dont see why you need to attach these elements to the app. Looks repetitive.